langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -317,6 +449,11 @@ public isolated function children(xml x) returns xml = @java:Method {
 #
 # This is equivalent to `children(x).elements(nm)`.
 #
+# ```ballerina
+# xml books = xml `<book><title>Java</title></book><book><title>Ballerina</title></book>`;
+# books.elementChildren("title") ⇒ <title>Java</title><title>Ballerina</title>
```

Comment:
Shall we add an example with `nm` being () as well

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/39130/commits/3a7088c41159f3c56a37443fa2036905f399eaa5